guides/common/modules/proc_configuring-the-base-operating-system-with-offline-repositories-in-rhel-8.adoc

```diff
@@ -0,0 +1,70 @@
+[id="configuring-the-base-operating-system-with-offline-repositories-in-rhel-8_{context}"]
+= Configuring the Base Operating System with Offline Repositories in RHEL 8
+
+Use this procedure to configure offline repositories for the {RHEL} 8 and {ProjectName} ISO images.
+
+.Procedure
+
+. Create a directory to serve as the mount point for the ISO file corresponding to the base operating system's version.
++
+[options="nowrap"]
+----
+# mkdir /media/rhel8
+----
+
+. Mount the ISO image for {RHEL} to the mount point.
++
+[options="nowrap" subs="+quotes"]
+----
+# mount -o loop _rhel8-DVD_.iso /media/rhel8
+----
++
+. To copy the ISO file's repository data file and change permissions, enter:
++
+[options="nowrap"]
+----
+# cp /media/rhel8/media.repo /etc/yum.repos.d/rhel8.repo
+# chmod u+w /etc/yum.repos.d/rhel8.repo
+----
+
+. Edit the repository data file and add the `baseurl` directive.
++
+[options="nowrap"]
+----
+[RHEL8-BaseOS]
+name=Red Hat Enterprise Linux BaseOS
+mediaid=None
+metadata_expire=-1
+gpgcheck=0
+cost=500
```

Comment:
Where does this value come from?

Reply:
This value was given by the QA.

Reply:
I think it's just the default one in the `media.repo` file on the ISO